lib/ecstatic/defaults.json

```diff
@@ -8,6 +8,7 @@
   "cache": "max-age=3600",
   "cors": false,
   "gzip": true,
+  "brotli": true,
```

Comment:
I think this should be false by default. This is consistent with how I've rolled out new features in the past - put them behind a FF and then enable them permanently in the next major version if no complaints.

Reply:
I think that's good reasoning, I've set it to false now.